package.json

```diff
@@ -77,7 +79,7 @@
   "dependencies": {
     "babel-preset-es2015": "^6.18.0",
     "classnames": "^2.2.0",
-    "cozy-client-js": "^0.0.1",
+    "cozy-client-js": "https://github.com/cozy/cozy-client-js.git#99b935bff6b3c3b83c58ea0f975b90a13a44c570",
```

Comment:
I don't like this. We need to publish the correct dependency on npm and maybe add a note in the `README.md` to explain a workaround if needed.